App/Inreach.php

```diff
@@ -261,6 +271,10 @@ function build_geojson() {
 						continue;						
 					}
 					
+					//fuzz the coordinates per settigs
```

Comment:
This should be conditional based on `location_precision` (see comment below). No need to truncate if default used.

You could move this condition as well as the reading of the Setting to the `truncateCoordinates` function itself.